Ref/Top/instances.fpp

```diff
@@ -33,155 +33,127 @@ module Ref {
-  # you wish to use
-
-  instance tlmSend: Svc.TlmChan base id 0x0C00 \
-    queue size Default.QUEUE_SIZE \
-    stack size Default.STACK_SIZE \
-    priority 97
-
-#   instance tlmSend: Svc.TlmPacketizer base id 0x0C00 \
-#       queue size Default.QUEUE_SIZE \
-#       stack size Default.STACK_SIZE \
-#       priority 97
-
-  instance prmDb: Svc.PrmDb base id 0x0D00 \
+  instance prmDb: Svc.PrmDb base id 0x0A00 \
     queue size Default.QUEUE_SIZE \
     stack size Default.STACK_SIZE \
     priority 96
 
-  instance dpCat: Svc.DpCatalog base id 0x0E00 \
+  instance dpCat: Svc.DpCatalog base id 0x0B00 \
     queue size Default.QUEUE_SIZE \
     stack size Default.STACK_SIZE \
     priority 96
 
-  instance dpMgr: Svc.DpManager base id 0x0F00 \
+  instance dpMgr: Svc.DpManager base id 0x0C00 \
     queue size Default.QUEUE_SIZE \
     stack size Default.STACK_SIZE \
     priority 96
 
-  instance dpWriter: Svc.DpWriter base id 0x1000 \
+  instance dpWriter: Svc.DpWriter base id 0x0D00 \
     queue size Default.QUEUE_SIZE \
     stack size Default.STACK_SIZE \
     priority 96
 
   # ComQueue has a deeper queue to be resilient to spikes in com throughput
-  instance comQueue: Svc.ComQueue base id 0x1100 \
+  instance comQueue: Svc.ComQueue base id 0x0E00 \
       queue size 50 \
       stack size Default.STACK_SIZE \
       priority 100
 
-
-  instance typeDemo: Ref.TypeDemo base id 0x1200
+  instance typeDemo: Ref.TypeDemo base id 0x0F00
 
   # ----------------------------------------------------------------------
   # Queued component instances
   # ----------------------------------------------------------------------
 
-  instance $health: Svc.Health base id 0x2000 \
+  instance $health: Svc.Health base id 0x1000 \
     queue size 25
 
-  instance SG1: Ref.SignalGen base id 0x2100 \
+  instance SG1: Ref.SignalGen base id 0x1100 \
     queue size Default.QUEUE_SIZE
 
-  instance SG2: Ref.SignalGen base id 0x2200 \
+  instance SG2: Ref.SignalGen base id 0x1200 \
     queue size Default.QUEUE_SIZE
 
-  instance SG3: Ref.SignalGen base id 0x2300 \
+  instance SG3: Ref.SignalGen base id 0x1300 \
     queue size Default.QUEUE_SIZE
 
-  instance SG4: Ref.SignalGen base id 0x2400 \
+  instance SG4: Ref.SignalGen base id 0x1400 \
     queue size Default.QUEUE_SIZE
 
-  instance SG5: Ref.SignalGen base id 0x2500 \
+  instance SG5: Ref.SignalGen base id 0x1500 \
     queue size Default.QUEUE_SIZE
 
-  instance sendBuffComp: Ref.SendBuff base id 0x2600 \
+  instance sendBuffComp: Ref.SendBuff base id 0x1600 \
     queue size Default.QUEUE_SIZE
   
-
-
   # ----------------------------------------------------------------------
   # Passive component instances
   # ----------------------------------------------------------------------
 
   @ Communications driver. May be swapped with other comm drivers like UART
-  instance comDriver: Drv.TcpClient base id 0x4000
+  instance comDriver: Drv.TcpClient base id 0x1700
```

Comment:
I believe we used to leave some room in between on purpose, to be able to add components easily. Arguable 2600 -> 4000 was too much.